frontend/src/cards/BarChartCard.tsx

```diff
@@ -65,14 +69,51 @@ function BarChartCardWithKey(props: BarChartCardProps) {
     props.variableConfig.metrics
   ).filter((metricConfig) => VALID_METRIC_TYPES.includes(metricConfig.type));
 
+  function CardTitle() {
+    const [anchorEl, handleClick, handleClose, open] = usePopover();
+
+    return (
+      <>
+        <Popover
+          open={open}
+          anchorEl={anchorEl}
+          onClose={handleClose}
+          anchorOrigin={{
+            vertical: "bottom",
+            horizontal: "center",
+          }}
+          transformOrigin={{
+            vertical: "top",
+            horizontal: "center",
+          }}
+        >
+          <div className={styles.CardTitlePopover}>
+            A <b>disparity</b> is defined by when if a health outcome is seen to
+            a greater or lesser extent between populations.{" "}
+            <a href="/">Learn more</a>
```

Comment:
Is this supposed to just link back to the main page?

Reply:
Dunno. Made a note to ask